CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ endif()
 
 option(USE_GL_ES "Use OpenGL ES instead of regular OpenGL" OFF)
 option(WARNINGS_AS_ERRORS "Treat compiler warnings as errors" ON)
-
+option(BUILD_BENCHMARKS "Build benchmarks" OFF)
```

Comment:
Slight issue here, the flags don't agree. One's called BUILD_BENCHMARKS the other at line 98 checks for RIGEL_BUILD_BENCHMARKS

Reply:
Ah sorry, forgot to change the other one!